app/views/about.html

```diff
@@ -1,41 +1,93 @@
+  </div>
+  
   <form name="vm.form" novalidate>
     <div class="breadcrumb">
       <h4>TEST CUSTOM FORM</h4>
     </div>
-       <tabset>
-         <tab ng-repeat="tab in vm.tabs"
-              heading="{{tab.title}}"
-              active="tab.active"
-              disable="tab.disabled"
-              select="vm.tabSelected($index)">
-           <formly-form model="tab.form.model" fields="tab.form.fields" form="vm.form" options="tab.form.options">
-           </formly-form>
-         </tab>
-       </tabset>
-       <button type="submit" class="btn btn-primary submit-button" ng-click="vm.onSubmit()">{{vm.submitLabel}}</button>
-       <button type="button" class="btn btn-default" ng-click="vm.resetAllForms()">Cancel</button>
+    <tabset>
+      <tab ng-repeat="tab in vm.tabs" heading="{{tab.title}}" active="tab.active" disable="tab.disabled" select="vm.tabSelected($index)">
+        <formly-form model="tab.form.model" fields="tab.form.fields" form="vm.form" options="tab.form.options">
+        </formly-form>
+      </tab>
+    </tabset>
+    <button type="submit" class="btn btn-primary submit-button" ng-click="vm.onSubmit()">{{vm.submitLabel}}</button>
+    <button type="button" class="btn btn-default" ng-click="vm.resetAllForms()">Cancel</button>
 
-     </form>
-     <hr />
-     <h2>Model Value</h2>
+  </form>
+  <hr />
+  <h2>Model Value</h2>
   <pre>{{vm.model | json}}</pre>
-     <!-- <h2>Tabs <small>(note, functions are not shown)</small></h2>
+
+  <hr />
+  <h2>Question Map</h2>
+  <pre>{{vm.questionMap | json}}</pre>
+
+  <!-- <h2>Tabs <small>(note, functions are not shown)</small></h2>
      <pre>{{vm.originalTabs | json}}</pre>
      <h2>Form Fields</h2>
      <pre ng-repeat="tab in vm.tabs">{{tab.form.fields | json}}</pre> -->
 
-     <!-- important scripts Section -->
-     <!-- Put custom templates here -->
-     <script type="text/ng-template" id="error-messages.html">
-           <formly-transclude></formly-transclude>
-           <div class="error-messages" ng-messages="fc.$error" ng-if="form.$submitted || options.formControl.$touched">
-             <div class="message" ng-message="{{::name}}" ng-repeat="(name, message) in ::options.validation.messages" style="color:red;">
-               {{message(fc.$viewValue, fc.$modelValue, this)}}
-             </div>
-           </div>
+  <!-- important scripts Section -->
+  <!-- Put custom templates here -->
+  <script type="text/ng-template" id="error-messages.html">
+    <formly-transclude></formly-transclude>
+    <div class="error-messages" ng-messages="fc.$error" ng-if="form.$submitted || options.formControl.$touched">
+      <div class="message" ng-message="{{::name}}" ng-repeat="(name, message) in ::options.validation.messages" style="color:red;">
+        {{message(fc.$viewValue, fc.$modelValue, this)}}
+      </div>
+    </div>
+
+  </script>
 
-     </script>
-</div>
+  <!-- Error summary template -->
+  <script type="text/ng-template" id="formly-error-summary.html">
```

Comment:
I also thought of the same thing.
I did this for two reasons:
1. It will allow other developers to have custom templates to the directive(in this case formly-error-simmary-directive). This allows them to change it the way the would like to have them displayed. 
2. This seems  to be formly approach (from examples).

Just like formly did, we can provide examples on how to use this, (in this case the example in the about view)